database/migrations/2021_04_05_193814_create_posts_table.php

```diff
@@ -17,7 +17,7 @@ public function up()
           $table ->increments('id');
           $table -> string('title');
           $table -> mediumText('body');
```

Comment:
```suggestion
          $table->mediumText('body');
```
Spaties horen hier niet.